src/main/java/org/broadinstitute/hellbender/engine/dataflow/DataflowCommandLineProgram.java

```diff
@@ -131,19 +138,24 @@ public static String getRunnerTypeName(String runnerSimpleName) {
             shortName = "numWorkers", fullName = "numWorkers", optional=true)
     protected int numWorkers = 0;
 
+    @Argument(doc = "The Google Compute Engine machine type that Dataflow will use when spinning up worker VMs",
+         fullName = "workerMachineType", optional=true)
+    protected String workerMachineType = "n1-standard-1";
+
     @Argument(fullName = "sparkMaster", doc="URL of the Spark Master to submit jobs to when using the Spark pipeline runner.", optional = true)
     protected String sparkMaster;
 
     @Override
-    protected String[] customCommandLineValidation(){
-        if((runnerType == PipelineRunnerType.BLOCKING || runnerType == PipelineRunnerType.NONBLOCKING)
-            && (projectID == null || stagingLocation==null)){
+    protected String[] customCommandLineValidation() {
+        if ((runnerType == PipelineRunnerType.BLOCKING || runnerType == PipelineRunnerType.NONBLOCKING)
+            && (projectID == null || stagingLocation==null)) {
             throw new UserException.CommandLineException(String.format("Non local dataflow execution requires project " +
-                    "and staging to be set. project:%s id:%s.",projectID, stagingLocation));
+                    "and staging to be set. project:%s staging location:%s.",projectID, stagingLocation));
         }
         return null;
     }
 
+    public GenomicsFactory.OfflineAuth offlineAuth;
```

Comment:
OK. I removed it instead, as now we can get the authentication info from the pipeline options. This is a lot more convenient.